### ISSUE TITLE
Small correction in "Web Application Lifecycle" section.

### ISSUE THREAD

**Issue by DmitriGit** [eclipse-ee4j/jakartaee-tutorial#255]
Hello, 

the inacurracy is small, but should be corrected, I think.

In the section "Web Application Lifecycle" (https://eclipse-ee4j.github.io/jakartaee-tutorial/#web-application-lifecycle) there is a phrase: 
"_Developing web component code is covered in the later chapters. Steps 2 through **4** are expanded on in the following sections and illustrated with a Hello, World–style, presentation-oriented application._"

Later you can read: 
"_The applications are used to illustrate tasks involved in packaging, **deploying**, configuring, **and running** an application that contains web components._"

And indeed, if you look forward in the chapter, these examples expand NON ONLY step 2 through 4, but also step 5 and step 6 (see "Packaging and Deploying the hello1 Web Module" and "Running the Deployed hello1 Web Module" sections of the same chapter).

So, in conclusion, the phrase should contain "**Steps 2 through 6**" ("6" instead of "4"):
"_Developing web component code is covered in the later chapters. Steps 2 through **6** are expanded on in the following sections and illustrated with a Hello, World–style, presentation-oriented application._"

Thank you,
Dmitri.

**Comment by DmitriGit** [comment]
Hello,

do you want I transform it in PR too the same as 257?

Thank you,
Dmitri.

**Comment by arjantijms** [comment]
Yes, please ;)
